CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 Note the first digit of every adapter version corresponds to the major version of the Chartboost Mediation SDK compatible with that adapter. 
 Adapters are compatible with any Chartboost Mediation SDK version within that major version.
 
+### 4.6.2.0.3
+- Removes a print stack trace when the user consent was being set before the HyprMX SDK was initialized.
```

Comment:
I've found this formula helps when writing changelog entries:

[Action] --> [Impact] --> [Context]

So something like:

```
Only set user consent for HyprMX if their SDK has initialized. This prevents ___ from happening.
```

Reply:
How's this @chauduyphanvu ?

>Only set user consent for HyprMX if their SDK has initialized. This prevents a stack trace from being printed.

Reply:
Is it just printing the stack trace or is there something more? I mean, at its core it's a caught exception (and the stack trace printing is a byproduct) so we can say ` ... to prevent NameOfException from happening`.

Reply:
I can add the name of the exception. Good call.

Reply:
Commit pushed. 🎉